CMakeLists.txt

```diff
@@ -20,18 +20,23 @@ set (CMAKE_PROJECT_VERSION_MAJOR "0")
 set (CMAKE_PROJECT_VERSION_MINOR "1")
 set (CMAKE_PROJECT_VERSION_PATCH "0")
 
+option (RUN_TEST
+  "If false, build libsxg without building tests. \
+  Building test for libsxg requires more recent version of cmake (>= 3.13) \
+  and a fuzzer supported by clang++. When you use g++ or other compilers, \
```

Comment:
"When you use other compliers, such as g++,"

Reply:
Done.